neuraxle/steps/output_handlers.py

```diff
@@ -132,15 +134,37 @@ def handle_inverse_transform(self, data_container: DataContainer, context: Execu
 
     def _set_expected_outputs(self, data_container, new_expected_outputs_data_container) -> DataContainer:
         if len(data_container.data_inputs) != len(data_container.expected_outputs):
-            raise AssertionError('OutputTransformerWrapper: Found different len for data inputs, and expected outputs. Please return the same the same amount of data inputs, and expected outputs.')
+            raise AssertionError(
+                'OutputTransformerWrapper: Found different len for data inputs, and expected outputs. Please return the same the same amount of data inputs, and expected outputs.')
```

Comment:
```suggestion
                'OutputTransformerWrapper: Found different len for data inputs, and expected outputs. '
                'Please return the same the same amount of data inputs, and expected outputs, '
                'or otherwise create your own handler methods to do more funky things.')
```